src/ESM.sol

```diff
@@ -70,12 +70,18 @@ contract ESM {
 
         end.cage();
         if (proxy != address(0)) {
-            VatLike(end.vat()).deny(proxy);
+            DenyLike(end.vat()).deny(proxy);
         }
 
         emit Fire();
     }
 
+    function deny(address target) external {
+        require(Sum >= min,  "ESM/min-not-reached");
+
+        DenyLike(target).deny(proxy);
+    }
+
```

Comment:
The only thing is we would need to add more authorities over the `esm`.

Reply:
Rather, give the ESM authority to access more contracts. But that can be done on an as-needed basis.

Reply:
Yeah that was referring too. Thanks for the correct rephrasing :)